src/main/java/duke/command/FindCommand.java

```diff
@@ -0,0 +1,18 @@
+package duke.command;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import duke.exception.DukeException;
+
+public class FindCommand extends Command {
```

Comment:
I think it would be good to add javadoc comments here

Reply:
I'm not sure if it is needed here. But I think I should at least have a Javadoc at least in the constructor or execute().
After all one would know what FindCommand probably is from inheritance but one wouldn't know what it does on execution.